### ISSUE TITLE
Running python example "GettingStarted" with flink-sql-connector-kinesis_2.12-1.13.2 and amazon-kinesis-sql-connector-flink-2.3.0

### ISSUE THREAD

**Issue by ajaen4** [aws-samples/amazon-kinesis-data-analytics-java-examples#20]
When running the getting started (python) the application does not start in KDA.

I have read [this](https://github.com/aws-samples/amazon-kinesis-data-analytics-java-examples/issues/4) issue where it is recommended to use the 'amazon-kinesis-sql-connector-flink-2.3.0.jar' as the jar file passed in the python script with no luck. The error I am getting is the following:

> Traceback (most recent call last):
>   File "/home/bluetab/Escritorio/OCS/Proyectos/new-nrt-dashboard-core/getting-started.py", line 154, in <module>
>     main()
>   File "/home/bluetab/Escritorio/OCS/Proyectos/new-nrt-dashboard-core/getting-started.py", line 143, in main
>     table_result = table_env.execute_sql("INSERT INTO {0} SELECT * FROM {1}"
>   File "/home/bluetab/Escritorio/OCS/Proyectos/venvs/new-nrt/lib/python3.8/site-packages/pyflink/table/table_environment.py", line 876, in execute_sql
>     return TableResult(self._j_tenv.executeSql(stmt))
>   File "/home/bluetab/Escritorio/OCS/Proyectos/venvs/new-nrt/lib/python3.8/site-packages/py4j/java_gateway.py", line 1285, in __call__
>     return_value = get_return_value(
>   File "/home/bluetab/Escritorio/OCS/Proyectos/venvs/new-nrt/lib/python3.8/site-packages/pyflink/util/exceptions.py", line 146, in deco
>     return f(*a, **kw)
>   File "/home/bluetab/Escritorio/OCS/Proyectos/venvs/new-nrt/lib/python3.8/site-packages/py4j/protocol.py", line 326, in get_return_value
>     raise Py4JJavaError(
> py4j.protocol.Py4JJavaError: An error occurred while calling o4.executeSql.
> : org.apache.flink.table.api.ValidationException: Unable to create a source for reading table 'default_catalog.default_database.input_table'.
> 
> Table options are:
> 
> 'aws.region'='eu-west-1'
> 'connector'='kinesis'
> 'format'='json'
> 'json.timestamp-format.standard'='ISO-8601'
> 'scan.stream.initpos'='LATEST'
> 'sink.partitioner-field-delimiter'=';'
> 'sink.producer.collection-max-count'='100'
> 'stream'='input-kinesis-apache-flink'
>         at org.apache.flink.table.factories.FactoryUtil.createTableSource(FactoryUtil.java:150)
>         at org.apache.flink.table.planner.plan.schema.CatalogSourceTable.createDynamicTableSource(CatalogSourceTable.java:116)
>         at org.apache.flink.table.planner.plan.schema.CatalogSourceTable.toRel(CatalogSourceTable.java:82)
>         at org.apache.calcite.sql2rel.SqlToRelConverter.toRel(SqlToRelConverter.java:3585)
>         at org.apache.calcite.sql2rel.SqlToRelConverter.convertIdentifier(SqlToRelConverter.java:2507)
>         at org.apache.calcite.sql2rel.SqlToRelConverter.convertFrom(SqlToRelConverter.java:2144)
>         at org.apache.calcite.sql2rel.SqlToRelConverter.convertFrom(SqlToRelConverter.java:2093)
>         at org.apache.calcite.sql2rel.SqlToRelConverter.convertFrom(SqlToRelConverter.java:2050)
>         at org.apache.calcite.sql2rel.SqlToRelConverter.convertSelectImpl(SqlToRelConverter.java:663)
>         at org.apache.calcite.sql2rel.SqlToRelConverter.convertSelect(SqlToRelConverter.java:644)
>         at org.apache.calcite.sql2rel.SqlToRelConverter.convertQueryRecursive(SqlToRelConverter.java:3438)
>         at org.apache.calcite.sql2rel.SqlToRelConverter.convertQuery(SqlToRelConverter.java:570)
>         at org.apache.flink.table.planner.calcite.FlinkPlannerImpl.org$apache$flink$table$planner$calcite$FlinkPlannerImpl$$rel(FlinkPlannerImpl.scala:177)
>         at org.apache.flink.table.planner.calcite.FlinkPlannerImpl.rel(FlinkPlannerImpl.scala:169)
>         at org.apache.flink.table.planner.operations.SqlToOperationConverter.toQueryOperation(SqlToOperationConverter.java:1057)
>         at org.apache.flink.table.planner.operations.SqlToOperationConverter.convertSqlQuery(SqlToOperationConverter.java:1026)
>         at org.apache.flink.table.planner.operations.SqlToOperationConverter.convert(SqlToOperationConverter.java:301)
>         at org.apache.flink.table.planner.operations.SqlToOperationConverter.convertSqlInsert(SqlToOperationConverter.java:639)
>         at org.apache.flink.table.planner.operations.SqlToOperationConverter.convert(SqlToOperationConverter.java:290)
>         at org.apache.flink.table.planner.delegation.ParserImpl.parse(ParserImpl.java:101)
>         at org.apache.flink.table.api.internal.TableEnvironmentImpl.executeSql(TableEnvironmentImpl.java:736)
>         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
>         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
>         at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
>         at java.base/java.lang.reflect.Method.invoke(Method.java:566)
>         at org.apache.flink.api.python.shaded.py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:244)
>         at org.apache.flink.api.python.shaded.py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:357)
>         at org.apache.flink.api.python.shaded.py4j.Gateway.invoke(Gateway.java:282)
>         at org.apache.flink.api.python.shaded.py4j.commands.AbstractCommand.invokeMethod(AbstractCommand.java:132)
>         at org.apache.flink.api.python.shaded.py4j.commands.CallCommand.execute(CallCommand.java:79)
>         at org.apache.flink.api.python.shaded.py4j.GatewayConnection.run(GatewayConnection.java:238)
>         at java.base/java.lang.Thread.run(Thread.java:829)
> Caused by: java.lang.NoSuchMethodError: 'org.apache.flink.table.catalog.CatalogTable org.apache.flink.table.factories.DynamicTableFactory$Context.getCatalogTable()'
>         at software.amazon.kinesis.connectors.flink.table.KinesisDynamicTableFactory.createDynamicTableSource(KinesisDynamicTableFactory.java:66)
>         at org.apache.flink.table.factories.FactoryUtil.createTableSource(FactoryUtil.java:147)
>         ... 31 more

When running the same code but with the original .jar file ('flink-sql-connector-kinesis_2.12-1.13.2') used in the official [documentation](https://docs.aws.amazon.com/kinesisanalytics/latest/java/gs-python-createapp.html)  . I get to run the application but it ends up failing with the following error:

> orio/OCS/Proyectos/new-nrt-dashboard-core/getting-started.py
> WARNING: An illegal reflective access operation has occurred
> WARNING: Illegal reflective access by org.apache.flink.api.java.ClosureCleaner (file:/home/bluetab/Escritorio/OCS/Proyectos/venvs/new-nrt/lib/python3.8/site-packages/pyflink/lib/flink-dist_2.11-1.14.4.jar) to field java.util.Collections$SingletonList.serialVersionUID
> WARNING: Please consider reporting this to the maintainers of org.apache.flink.api.java.ClosureCleaner
> WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
> WARNING: All illegal access operations will be denied in a future release
> Traceback (most recent call last):
>   File "/home/bluetab/Escritorio/OCS/Proyectos/new-nrt-dashboard-core/getting-started.py", line 154, in <module>
>     main()
>   File "/home/bluetab/Escritorio/OCS/Proyectos/new-nrt-dashboard-core/getting-started.py", line 147, in main
>     table_result.wait()
>   File "/home/bluetab/Escritorio/OCS/Proyectos/venvs/new-nrt/lib/python3.8/site-packages/pyflink/table/table_result.py", line 76, in wait
>     get_method(self._j_table_result, "await")()
>   File "/home/bluetab/Escritorio/OCS/Proyectos/venvs/new-nrt/lib/python3.8/site-packages/py4j/java_gateway.py", line 1285, in __call__
>     return_value = get_return_value(
>   File "/home/bluetab/Escritorio/OCS/Proyectos/venvs/new-nrt/lib/python3.8/site-packages/pyflink/util/exceptions.py", line 146, in deco
>     return f(*a, **kw)
>   File "/home/bluetab/Escritorio/OCS/Proyectos/venvs/new-nrt/lib/python3.8/site-packages/py4j/protocol.py", line 326, in get_return_value
>     raise Py4JJavaError(
> py4j.protocol.Py4JJavaError: An error occurred while calling o90.await.
> : java.util.concurrent.ExecutionException: org.apache.flink.table.api.TableException: Failed to wait job finish
>         at java.base/java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:395)
>         at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:1999)
>         at org.apache.flink.table.api.internal.TableResultImpl.awaitInternal(TableResultImpl.java:129)
>         at org.apache.flink.table.api.internal.TableResultImpl.await(TableResultImpl.java:92)
>         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
>         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
>         at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
>         at java.base/java.lang.reflect.Method.invoke(Method.java:566)
>         at org.apache.flink.api.python.shaded.py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:244)
>         at org.apache.flink.api.python.shaded.py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:357)
>         at org.apache.flink.api.python.shaded.py4j.Gateway.invoke(Gateway.java:282)
>         at org.apache.flink.api.python.shaded.py4j.commands.AbstractCommand.invokeMethod(AbstractCommand.java:132)
>         at org.apache.flink.api.python.shaded.py4j.commands.CallCommand.execute(CallCommand.java:79)
>         at org.apache.flink.api.python.shaded.py4j.GatewayConnection.run(GatewayConnection.java:238)
>         at java.base/java.lang.Thread.run(Thread.java:829)
> Caused by: org.apache.flink.table.api.TableException: Failed to wait job finish
>         at org.apache.flink.table.api.internal.InsertResultIterator.hasNext(InsertResultIterator.java:56)
>         at org.apache.flink.table.api.internal.TableResultImpl$CloseableRowIteratorWrapper.hasNext(TableResultImpl.java:370)
>         at org.apache.flink.table.api.internal.TableResultImpl$CloseableRowIteratorWrapper.isFirstRowReady(TableResultImpl.java:383)
>         at org.apache.flink.table.api.internal.TableResultImpl.lambda$awaitInternal$1(TableResultImpl.java:116)
>         at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1736)
>         at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
>         at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
>         ... 1 more
> Caused by: java.util.concurrent.ExecutionException: org.apache.flink.runtime.client.JobExecutionException: Job execution failed.
>         at java.base/java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:395)
>         at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:1999)
>         at org.apache.flink.table.api.internal.InsertResultIterator.hasNext(InsertResultIterator.java:54)
>         ... 7 more
> Caused by: org.apache.flink.runtime.client.JobExecutionException: Job execution failed.
>         at org.apache.flink.runtime.jobmaster.JobResult.toJobExecutionResult(JobResult.java:144)
>         at org.apache.flink.runtime.minicluster.MiniClusterJobClient.lambda$getJobExecutionResult$3(MiniClusterJobClient.java:137)
>         at java.base/java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:642)
>         at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
>         at java.base/java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2073)
>         at org.apache.flink.runtime.rpc.akka.AkkaInvocationHandler.lambda$invokeRpc$1(AkkaInvocationHandler.java:258)
>         at java.base/java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:859)
>         at java.base/java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:837)
>         at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
>         at java.base/java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2073)
>         at org.apache.flink.util.concurrent.FutureUtils.doForward(FutureUtils.java:1389)
>         at org.apache.flink.runtime.concurrent.akka.ClassLoadingUtils.lambda$null$1(ClassLoadingUtils.java:93)
>         at org.apache.flink.runtime.concurrent.akka.ClassLoadingUtils.runWithContextClassLoader(ClassLoadingUtils.java:68)
>         at org.apache.flink.runtime.concurrent.akka.ClassLoadingUtils.lambda$guardCompletionWithContextClassLoader$2(ClassLoadingUtils.java:92)
>         at java.base/java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:859)
>         at java.base/java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:837)
>         at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
>         at java.base/java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2073)
>         at org.apache.flink.runtime.concurrent.akka.AkkaFutureUtils$1.onComplete(AkkaFutureUtils.java:47)
>         at akka.dispatch.OnComplete.internal(Future.scala:300)
>         at akka.dispatch.OnComplete.internal(Future.scala:297)
>         at akka.dispatch.japi$CallbackBridge.apply(Future.scala:224)
>         at akka.dispatch.japi$CallbackBridge.apply(Future.scala:221)
>         at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:60)
>         at org.apache.flink.runtime.concurrent.akka.AkkaFutureUtils$DirectExecutionContext.execute(AkkaFutureUtils.java:65)
>         at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:68)
>         at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1(Promise.scala:284)
>         at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1$adapted(Promise.scala:284)
>         at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:284)
>         at akka.pattern.PromiseActorRef.$bang(AskSupport.scala:621)
>         at akka.pattern.PipeToSupport$PipeableFuture$$anonfun$pipeTo$1.applyOrElse(PipeToSupport.scala:24)
>         at akka.pattern.PipeToSupport$PipeableFuture$$anonfun$pipeTo$1.applyOrElse(PipeToSupport.scala:23)
>         at scala.concurrent.Future.$anonfun$andThen$1(Future.scala:532)
>         at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:29)
>         at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:29)
>         at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:60)
>         at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:63)
>         at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:100)
>         at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
>         at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:81)
>         at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:100)
>         at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:49)
>         at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:48)
>         at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
>         at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
>         at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
>         at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
>         at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)
> Caused by: org.apache.flink.runtime.JobException: Recovery is suppressed by NoRestartBackoffTimeStrategy
>         at org.apache.flink.runtime.executiongraph.failover.flip1.ExecutionFailureHandler.handleFailure(ExecutionFailureHandler.java:138)
>         at org.apache.flink.runtime.executiongraph.failover.flip1.ExecutionFailureHandler.getFailureHandlingResult(ExecutionFailureHandler.java:82)
>         at org.apache.flink.runtime.scheduler.DefaultScheduler.handleTaskFailure(DefaultScheduler.java:252)
>         at org.apache.flink.runtime.scheduler.DefaultScheduler.maybeHandleTaskFailure(DefaultScheduler.java:242)
>         at org.apache.flink.runtime.scheduler.DefaultScheduler.updateTaskExecutionStateInternal(DefaultScheduler.java:233)
>         at org.apache.flink.runtime.scheduler.SchedulerBase.updateTaskExecutionState(SchedulerBase.java:684)
>         at org.apache.flink.runtime.scheduler.SchedulerNG.updateTaskExecutionState(SchedulerNG.java:79)
>         at org.apache.flink.runtime.jobmaster.JobMaster.updateTaskExecutionState(JobMaster.java:444)
>         at jdk.internal.reflect.GeneratedMethodAccessor48.invoke(Unknown Source)
>         at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
>         at java.base/java.lang.reflect.Method.invoke(Method.java:566)
>         at org.apache.flink.runtime.rpc.akka.AkkaRpcActor.lambda$handleRpcInvocation$1(AkkaRpcActor.java:316)
>         at org.apache.flink.runtime.concurrent.akka.ClassLoadingUtils.runWithContextClassLoader(ClassLoadingUtils.java:83)
>         at org.apache.flink.runtime.rpc.akka.AkkaRpcActor.handleRpcInvocation(AkkaRpcActor.java:314)
>         at org.apache.flink.runtime.rpc.akka.AkkaRpcActor.handleRpcMessage(AkkaRpcActor.java:217)
>         at org.apache.flink.runtime.rpc.akka.FencedAkkaRpcActor.handleRpcMessage(FencedAkkaRpcActor.java:78)
>         at org.apache.flink.runtime.rpc.akka.AkkaRpcActor.handleMessage(AkkaRpcActor.java:163)
>         at akka.japi.pf.UnitCaseStatement.apply(CaseStatements.scala:24)
>         at akka.japi.pf.UnitCaseStatement.apply(CaseStatements.scala:20)
>         at scala.PartialFunction.applyOrElse(PartialFunction.scala:123)
>         at scala.PartialFunction.applyOrElse$(PartialFunction.scala:122)
>         at akka.japi.pf.UnitCaseStatement.applyOrElse(CaseStatements.scala:20)
>         at scala.PartialFunction$OrElse.applyOrElse(PartialFunction.scala:171)
>         at scala.PartialFunction$OrElse.applyOrElse(PartialFunction.scala:172)
>         at scala.PartialFunction$OrElse.applyOrElse(PartialFunction.scala:172)
>         at akka.actor.Actor.aroundReceive(Actor.scala:537)
>         at akka.actor.Actor.aroundReceive$(Actor.scala:535)
>         at akka.actor.AbstractActor.aroundReceive(AbstractActor.scala:220)
>         at akka.actor.ActorCell.receiveMessage(ActorCell.scala:580)
>         at akka.actor.ActorCell.invoke(ActorCell.scala:548)
>         at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:270)
>         at akka.dispatch.Mailbox.run(Mailbox.scala:231)
>         at akka.dispatch.Mailbox.exec(Mailbox.scala:243)
>         ... 5 more
> Caused by: java.lang.NoSuchMethodError: 'org.apache.flink.metrics.MetricGroup org.apache.flink.api.common.functions.RuntimeContext.getMetricGroup()'
>         at org.apache.flink.streaming.connectors.kinesis.internals.KinesisDataFetcher.<init>(KinesisDataFetcher.java:416)
>         at org.apache.flink.streaming.connectors.kinesis.internals.KinesisDataFetcher.<init>(KinesisDataFetcher.java:365)
>         at org.apache.flink.streaming.connectors.kinesis.FlinkKinesisConsumer.createFetcher(FlinkKinesisConsumer.java:536)
>         at org.apache.flink.streaming.connectors.kinesis.FlinkKinesisConsumer.run(FlinkKinesisConsumer.java:308)
>         at org.apache.flink.streaming.api.operators.StreamSource.run(StreamSource.java:110)
>         at org.apache.flink.streaming.api.operators.StreamSource.run(StreamSource.java:67)
>         at org.apache.flink.streaming.runtime.tasks.SourceStreamTask$LegacySourceFunctionThread.run(SourceStreamTask.java:323)
>         Suppressed: java.lang.NullPointerException
>                 at org.apache.flink.streaming.connectors.kinesis.FlinkKinesisConsumer.close(FlinkKinesisConsumer.java:421)
>                 at org.apache.flink.api.common.functions.util.FunctionUtils.closeFunction(FunctionUtils.java:41)
>                 at org.apache.flink.streaming.api.operators.AbstractUdfStreamOperator.close(AbstractUdfStreamOperator.java:114)
>                 at org.apache.flink.streaming.api.operators.StreamSource.close(StreamSource.java:124)
>                 at org.apache.flink.streaming.runtime.tasks.StreamOperatorWrapper.close(StreamOperatorWrapper.java:141)
>                 at org.apache.flink.streaming.runtime.tasks.RegularOperatorChain.closeAllOperators(RegularOperatorChain.java:127)
>                 at org.apache.flink.streaming.runtime.tasks.StreamTask.closeAllOperators(StreamTask.java:1035)
>                 at org.apache.flink.streaming.runtime.tasks.StreamTask.runAndSuppressThrowable(StreamTask.java:1021)
>                 at org.apache.flink.streaming.runtime.tasks.StreamTask.cleanUp(StreamTask.java:928)
>                 at org.apache.flink.runtime.taskmanager.Task.lambda$restoreAndInvoke$0(Task.java:940)
>                 at org.apache.flink.runtime.taskmanager.Task.runWithSystemExitMonitoring(Task.java:958)
>                 at org.apache.flink.runtime.taskmanager.Task.restoreAndInvoke(Task.java:940)
>                 at org.apache.flink.runtime.taskmanager.Task.doRun(Task.java:766)
>                 at org.apache.flink.runtime.taskmanager.Task.run(Task.java:575)
>                 at java.base/java.lang.Thread.run(Thread.java:829)

When decompiling both .jars I see the following packages:
![image](https://user-images.githubusercontent.com/26901448/158795220-44b805d6-6075-4eb2-8c2f-3985aa88ef97.png)

From my point of view, it looks like the new version of the .jar (amazon-kinesis-sql-connector-flink-2.3.0) is now missing the org.apache.flink package which did appear in the old .jar (flink-sql-connector-kinesis_2.12-1.13.2) but in that case it should be getting a "ClassNotFoundException" instead of "NoSuchMethod"...

I am using apache-flink==1.14.4.

Thanks for your help!

**Comment by ajaen4** [comment]
After going down to apache-flink==1.13.2 and using the flink-sql-connector-kinesis_2.12-1.13.2.jar file (the other kept giving me a 'NoSuchMethodError') it started giving me another type of permissions error.

Moved to AWS instead of running it locally and after debugging it I found out I was missing some permissions that are not present in the policy (cloudwatch and kinesis list shards) present in [this](https://docs.aws.amazon.com/kinesisanalytics/latest/java/gs-python-createapp.html) documentation. Would be interesting to improve the documentation to be more detailed, specially being a getting started example.

All in all, it is now working. Closing issue.

**Comment by ajaen4** [comment]
After going down to apache-flink==1.13.2 and using the flink-sql-connector-kinesis_2.12-1.13.2.jar file (the other kept giving me a 'NoSuchMethodError') it started giving me another type of permissions error.

Moved to AWS instead of running it locally and after debugging it I found out I was missing some permissions that are not present in the policy (cloudwatch and kinesis list shards) present in [this](https://docs.aws.amazon.com/kinesisanalytics/latest/java/gs-python-createapp.html) documentation. Would be interesting to improve the documentation to be more detailed, specially being a getting started example.

All in all, it is now working. Closing issue.
